code/game/mecha/mecha_construction_paths.dm

```diff
@@ -1450,7 +1450,7 @@
 				holder.icon_state = "phazon16"
 			else
 				user.visible_message("[user] removes the bluespace crystal from the [holder].", "You remove the bluespace crystal from the [holder].")
-				new /obj/item/stack/ore/bluespace_crystal(get_turf(holder), new_amount = 5)
+				new /obj/item/stack/ore/bluespace_crystal(get_turf(holder))
```

Comment:
The stack code itself has new_amount as a param. Best to add it to the bluespace_crystal new

Reply:
`/obj/item/stack/New(loc, new_amount, merge = TRUE)` this goes all the way down the chain. Will add it going up.